src/Driver.ts

```diff
@@ -58,3 +57,21 @@ export const DRIVERS = {
   MEMORY: new Driver(new MemoryStorage()),
   COOKIE: new Driver(new CookieStorage())
 }
+
+export function determineDriver(defaultDriverType?: any): Driver {
+  let hasFallback = Array.isArray(defaultDriverType)
```

Comment:
If you check function/tests again, you will see that it preserves both single and multiple.  I made sure the change is backward compatible.